### DISCUSSION TITLE
14版这换了app图标，连iOS的都变不好看了，iOS和安卓全看起来显示有问题，

### DISCUSSION THREAD

**Discussion by wuhuining** [xiaohucode/yidaRule#96] [General]
![Uploading IMG_0088.jp
![IMG_0087](https://github.com/xiaohucode/yidaRule/assets/57199346/71366d7d-d6cf-436a-813b-e4ee29f904b6)
eg…]()

**Comment by ZHONG-ZHILIU** [comment]
我感觉小熊猫的图标挺好看的！

**Comment by mingaiki** [comment]
正文上边距为什么不能调整

**Comment by mingaiki** [comment]
您刚才发送的邮件已收到，谢谢您的来信。
